pkg/service/repair/plan.go

```diff
@@ -13,124 +13,138 @@ import (
 	"github.com/scylladb/scylla-manager/v3/pkg/util/slice"
 )
 
+type hkt = scyllaclient.HostKeyspaceTable
+
 // plan describes whole repair schedule and state.
 type plan struct {
 	Keyspaces []keyspacePlan
-	Idx       int // Idx of currently repaired keyspace
 
-	SkippedKeyspaces []string
+	Hosts            []string
 	MaxParallel      int
 	MaxHostIntensity map[string]Intensity
-	HostTableSize    map[scyllaclient.HostKeyspaceTable]int64
+	// Used for progress purposes
+	HostSize   map[hkt]int64
+	HostRanges map[hkt]int
 }
 
 func newPlan(ctx context.Context, target Target, client *scyllaclient.Client) (*plan, error) {
```

Comment:
This constructor is pretty big.

Is there a chance to either:
- update this method with comments explaining what step is it and what it's gonna do ...
- ... or (better) split this bit too big constructor to smaller pieces (functions) having self explanatory names ?